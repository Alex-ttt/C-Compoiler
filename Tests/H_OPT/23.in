void main(){
	int a = 4;
	while(0)
		a++;
	return;
}