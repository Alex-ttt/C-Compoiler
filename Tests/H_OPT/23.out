 Symbols Table:
   main  void function main(
  )
    {
      Statements:
      while () ..
      Condition:
        0
      Body:
        {
          Statements:
          a
          ++
        }
      Symbols Table:
              a int
    }
